htdocs/product/card.php

```diff
@@ -1649,29 +1649,37 @@
 
 						print '<script type="text/javascript">
 						$(document).ready(function() {
-							$("#field_mask, #mask_option").addClass("hideobject");
-							var preselect = $("#status_batch option:selected");
-							if (preselect !== "0") {
-								$("#field_mask, #mask_option").toggleClass("hideobject");
-							}
-							$("#status_batch").on("change", function () {
-								console.log("We change batch status");
+							$("#field_mask").parent().addClass("hideobject");
+							var preselect = document.getElementById("status_batch");';
+						if ($conf->global->PRODUCTBATCH_SN_USE_PRODUCT_MASKS)
```

Comment:
From v14, use 
if (getDolGlobalString('PRODUCTBATCH_SN_USE_PRODUCT_MASKS'))
instead of 
if ($conf->global->PRODUCTBATCH_SN_USE_PRODUCT_MASKS)
So it will avoid errors with php8

Reply:
sure thing. I suppose every $conf->global is concerned, even when it is not an if condition ?

Reply:
Yes. But this will be fixed step by step, commit after commit. This is a long term task.

Reply:
Done here, I didn’t plan on refurbishing the all product card, indeed, just did it in the section I was modifying in this particular PR